libs/components/forms/src/lib/modules/file-attachment/file-drop.component.scss

```diff
@@ -155,7 +155,13 @@ button.sky-file-drop {
 }
 
 .sky-file-drop-errors {
-  margin-bottom: var(--sky-margin-stacked-lg);
```

Comment:
Reverting this change to the CSS variable because the custom property is different across themes and that is an unintentional breaking change.